File: tests/bindings/dotnet/foo.Tests/ConstantTest.cs
using System;
using Xunit;
using foo;

namespace foo.Tests
{   
    public class ConstantTests
    {
        [Fact]
        public void ConstantTest()
        {
            Assert.Equal(1, SpecialValues.One);
            Assert.Equal(2, SpecialValues.Two);
        }
    }
}

File: tests/bindings/java/foo-tests/src/test/java/io/stepfunc/foo_test/ConstantTest.java
package io.stepfunc.foo_test;

import io.stepfunc.foo.SpecialValues;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class ConstantTest {
    @Test
    void specialValues() {
        assertThat(SpecialValues.ONE.byteValue()).isEqualTo((byte) 0x01);
        assertThat(SpecialValues.TWO.byteValue()).isEqualTo((byte) 0x02);
    }
}
